documentation/source/user_section/tutorials/registration-to-template/lumbar-registration/labeling.rst

```diff
@@ -0,0 +1,74 @@
+.. _manual-labeling-of-lumbar-landmarks:
+
+Adding landmark labels for template matching
+############################################
+
+Typically, registering an image to the PAM50 template involves using ``sct_label_vertebrae`` to automatically label vertebrae, then selecting 2 intervertebral disc labels to use for matching with the PAM50 template. However, using discs as registration landmarks presents a dilemma: How do we handle variability in the position of the cauda equinea relative to the L1-L2 disc?
+
+Notably, in the PAM50 template, the conus medullaris (i.e. the terminal end of the spinal cord) is aligned with the L1-L2 disc. However, for your subjects, the spinal cord may end above or below this point. So, if registration were based on disc landmarks alone, then the tapered region of the spinal cord may end up misaligned with the template.
+
+To correct for this, the PAM50 template provides a cauda equinea label (specifically, the conus medullaris) as a registration landmark. By creating a similar label in your subject data, you can align the terminal end of the spinal cord between your subject and the PAM50 template. However, this comes with the necessary tradeoff of potentially misaligning the discs.
+
+Deciding which landmarks to use (discs, conus medullaris)
+=========================================================
+
+Registration can be performed with either one or two labels. So, given the tradeoffs between the aligning the discs and/or the conus medullaris, the question becomes: Should registration be done with 1 disc label, 1 conus medullaris label, or a combination of the two?
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/master/lumbar-registration/label-comparison.png
+   :align: center
+
+Above is a comparison of sample results, demonstrating that registration will vary significantly depending on which landmarks are chosen. The "ideal result" will depend on the goals of your research. However, in most cases, you will want to stick with one approach and use it consistently throughout your research.
```

Comment:
I'm not sure which of the registration results are "ideal".

@jcohenadad initially recommended just the cauda equinea label initially, but then we had a second user choose both the cauda equinea _plus_ another landmark higher up in the cord. So, I included both techniques here for comparison.

Reply:
@jcohenadad suggests that the 60 label is hard to see

Reply:
I've made a handful of tweaks to improve readability:

Reply:
I've made a handful of tweaks here to improve readability:

![image](https://github.com/spinalcordtoolbox/spinalcordtoolbox/assets/16181459/0f5b90aa-412a-45cb-9613-49aff660b189)

- Higher contrast label colors
- Larger label text size
- Higher contrast arrows
- Using identical arrow locations for all 3 pictures (to make comparing/contrasting easier)
- Left-aligning the bottom text
- Using emojis to better clarify intent